docs/doctest.sh

```
@@ -0,0 +1,166 @@
+# This script runs documentation tests for markdown files in the docs directory.
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)